recipes/genbooster/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "genbooster" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Techtonique/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: cd1582ea4064f39041fc47771effad2b0232118c065ecf8f99a87b41d6083023
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py build_ext --inplace && {{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py < 39 or win]
+
+requirements:
+  build:
+    - rust >=1.70
+    - cargo-bundle-licenses
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}    
+  host:
+    - python
+    - pip
+    - setuptools
+    - maturin >=1.0.0
+    - numpy
+    - jax >=0.1.72  # [unix]
+    - jaxlib >=0.1.51  # [unix]
+    - matplotlib-base
+    - scikit-learn >=0.18.0
+    - pandas >=1.3.5
+    - joblib >=0.14.0
+    - pyvinecopulib >=0.6.6
+    - six >=1.16.0
+    - threadpoolctl >=2.0.0
+    - tqdm >=4.48.1
+    - requests
+    - statsmodels
+    - nnetsauce
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy >=0.19.0
+    - scikit-learn >=0.18.0
+    - matplotlib-base
+    - jax >=0.1.72  # [unix]
+    - jaxlib >=0.1.51  # [unix]
+    - pandas >=1.3.5
+    - joblib >=0.14.0
+    - pyvinecopulib >=0.6.6
+    - six >=1.16.0
+    - threadpoolctl >=2.0.0
+    - tqdm >=4.48.1
+    - requests
+    - statsmodels
+    - nnetsauce
+
+test:
+  requires:
+    - pytest
+    - numpy
+    - pandas
```

Comment:
You are not running the tests, can you make use of those and run it here? It is highly recommended for compiled extensions.

Reply:
Please add the pip check test and run the tests here, or remove the tests dependencies. The former is preferred b/c this is a compiled extension.